_src/static/sass/utilities/_mixins.scss

```diff
@@ -0,0 +1,468 @@
+        }
+    }
+}
+
+%button-secondary {
+    @extend %button;
+    background-color: $brand-secondary;
+
+    &:hover,
+    &:active,
+    &:focus {
+        background-color: $brand-secondary-lighter;
+        box-shadow: 0 2px 1px $shadow;
+    }
+
+    &.current,
+    &.active {
+        box-shadow: inset 1px 1px 2px $shadow;
+
+        &:hover,
+        &:active,
+        &:focus {
+              box-shadow: inset 1px 1px 1px $shadow;
+        }
+    }
+}
+
+%button-tertiary {
+    @extend %button;
+    background-color: $white;
+    border-color: $gray-darker;
+
+    &:hover,
+    &:active,
+    &:focus {
+        box-shadow: 0 2px 1px $shadow;
+    }
+
+    &.current,
+    &.active {
+        box-shadow: inset 1px 1px 2px $shadow;
+
+        &:hover,
+        &:active,
+        &:focus {
+              box-shadow: inset 1px 1px 1px $shadow;
+        }
+    }
+}
+
+
+/* ============================
+14. Wells
+---------------------------- */
+%well {
+    box-shadow: inset 0 1px 2px 1px $shadow;
+    padding: ($baseline * 0.75) $baseline;
+}
+
+%no-content {
+    @extend %well;
+    padding: ($baseline*2);
+    background-color: $gray-lightest;
+    text-align: center;
+    color: $gray;
+
+    .button,
+    .action {
+        @extend %button-secondary;
+        margin-left: $baseline;
+
+        .icon {
+            margin-right: ($baseline / 2);
+        }
+    }
+}
+
+.is-loading {
+    @include animation(fadeIn $timing-fast linear 1);
+    @extend %well;
+    padding: ($baseline * 1.5) $baseline;
+    background-color: $white;
+    font-size: %font-size;
+    text-align: center;
+    opacity: .6;
+
+    .spin {
+        @extend %anim-rotateCW;
+        display: inline-block;
+    }
+
+    .copy {
+        padding-left: ($baseline / 4);
+    }
+}
+
+.is-hidden {
+    display: none;
+}
+
+
+/* ============================
+15. Lists
+---------------------------- */
+%list-unstyled {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-indent: 0;
+
+    li,
+    dt,
+    dd {
+        margin: 0;
+        padding: 0;
+    }
+}
+
+%list-inline {
+    @extend %list-unstyled;
+
+    li,
+    dt,
+    dd {
+        display: inline-block;
+    }
+}
+
+
+/* ============================
+16. Icon fonts
+---------------------------- */
+%icon-font {
+    display: inline-block;
+    min-width: 1rem; // Should be 16px
+    min-height: 1rem; // Should be 16px
+    speak: none;
+}
+
+
+/* ============================
+17. Accessibility
+---------------------------- */
+%a11y-hide-text-image {
+    text-indent: 100%;
+    white-space: nowrap;
+    overflow: hidden;
+}
+
+%a11y-hide-text {
+    border: 0;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+}
+
+
+/* ============================
+18. Text Handling
+---------------------------- */
+%wrap-text {
+    word-wrap: break-word;
+}
+
+%truncate-text {
+    @include box-sizing(border-box);
```

Comment:
Please remove, this is not needed as Bourbon has deprecated this utility (its natively supported enough now!) and our wildcard reset will take care of this.